test/test_draw.py

```diff
@@ -1,4 +1,7 @@
 import unittest
+from Tkinter import Image
+
+""" Testing Image for a given imageSize=(640,480) . Checking whether the given points lies in the plane """
```

Comment:
Let's break the docstrings into multiple lines like this:

"""
Testing Image for a given imageSize=(640,480).
Checking whether the given points lies in the plane.
"""

So basically:
1. keep """ on its own line
2. keep the message under some number of columns, let's say 80. Though if it's 81 columns I don't care.